File: src/main/java/org/example/Java/Act10.java
package org.example.Java;

import java.util.Scanner;

public class Act10 {
    Scanner scanner = new Scanner(System.in);
    public String Temperatura() {
        System.out.println("Que quieres F o C ?");
        String option = scanner.nextLine();

        if (option.equals("F")) {
            System.out.println("Cuantos Celsius vas a tranformar?");
            Double Celsius = Double.parseDouble(scanner.nextLine());

            Double fahrenheit = Celsius * 1.8 + 32;

            return fahrenheit.toString();
        }else if (option.equals("C")) {
            System.out.println("Cuantos Fahrenheits vas a tranformar?");
            Double Fahrenheit = Double.parseDouble(scanner.nextLine());
            Double celsius = (Fahrenheit - 32)*(5/9);

            return celsius.toString();
        }
        return "no es una opcion";
    }
}

File: src/main/java/org/example/Kotlin/Act10.kt
package org.example.Kotlin

class Act10 {
    fun Temperatura(): String {
        println("Que quieres F o C ?")
        val option: String = readLine().toString()

        if (option == "F") {
            println("Cuantos Celsius vas a tranformar?")
            val Celsius: Double = readLine()!!.toDouble()

            val fahrenheit = Celsius * 1.8 + 32

            return fahrenheit.toString()
        } else if (option == "C") {
            println("Cuantos Fahrenheits vas a tranformar?")
            val Fahrenheit: Double = readLine()!!.toDouble()
            val celsius = (Fahrenheit - 32) * (5 / 9)

            return celsius.toString()
        }
        return "no es una opcion"
    }
}